"sbt consoleProject" does not work

## Minimized code


#### `build.sbt`

```scala
scalaVersion := "0.24.0-RC1"
```

#### `project/build.properties`

```
sbt.version=1.3.10
```

#### `project/plugins.sbt`

```scala
addSbtPlugin("ch.epfl.lamp" % "sbt-dotty" % "0.4.1")
```


run: `sbt consoleProject`

## Output

```
[error] (run-main-0) java.lang.NoClassDefFoundError: dotty/tools/repl/ReplDriver
[error] java.lang.NoClassDefFoundError: dotty/tools/repl/ReplDriver
[error] 	at xsbt.ConsoleInterface.run(ConsoleInterface.java:43)
[error] 	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[error] 	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
[error] 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[error] 	at java.lang.reflect.Method.invoke(Method.java:498)
[error] 	at sbt.internal.inc.AnalyzingCompiler.call(AnalyzingCompiler.scala:248)
[error] 	at sbt.internal.inc.AnalyzingCompiler.console(AnalyzingCompiler.scala:210)
[error] 	at sbt.Console.console0$1(Console.scala:48)
[error] 	at sbt.Console.$anonfun$apply$2(Console.scala:51)
[error] 	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
[error] 	at sbt.util.InterfaceUtil$$anon$1.get(InterfaceUtil.scala:10)
[error] 	at sbt.TrapExit$App.run(TrapExit.scala:257)
[error] 	at java.lang.Thread.run(Thread.java:748)
[error] Caused by: java.lang.ClassNotFoundException: dotty.tools.repl.ReplDriver
[error] 	at java.net.URLClassLoader.findClass(URLClassLoader.java:382)
[error] 	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
[error] 	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
[error] 	at xsbt.ConsoleInterface.run(ConsoleInterface.java:43)
[error] 	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[error] 	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
[error] 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[error] 	at java.lang.reflect.Method.invoke(Method.java:498)
[error] 	at sbt.internal.inc.AnalyzingCompiler.call(AnalyzingCompiler.scala:248)
[error] 	at sbt.internal.inc.AnalyzingCompiler.console(AnalyzingCompiler.scala:210)
[error] 	at sbt.Console.console0$1(Console.scala:48)
[error] 	at sbt.Console.$anonfun$apply$2(Console.scala:51)
[error] 	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
[error] 	at sbt.util.InterfaceUtil$$anon$1.get(InterfaceUtil.scala:10)
[error] 	at sbt.TrapExit$App.run(TrapExit.scala:257)
[error] 	at java.lang.Thread.run(Thread.java:748)
[error] Nonzero exit code: 1
[error] (consoleProject) Nonzero exit code: 1
```

## Expectation

--- a/sbt-dotty/src/dotty/tools/sbtplugin/DottyPlugin.scala
+++ b/sbt-dotty/src/dotty/tools/sbtplugin/DottyPlugin.scala
@@ -255,6 +255,10 @@ object DottyPlugin extends AutoPlugin {
         }
       }.value,
 
+      // Prevent the consoleProject task from using the Scala 3 compiler bridge
+      // The consoleProject must load the Scala 2.12 instance and the sbt classpath
+      consoleProject / scalaCompilerBridgeBinaryJar := None,
+
       // Needed for RCs publishing
       scalaBinaryVersion := {
         scalaVersion.value.split("[\\.-]").toList match {

